students/20th/hwisungkim/login.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style/login.css">
+    <script src="js/login.js" defer></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com"> 
+    <link href="https://fonts.googleapis.com/css2?family=Lobster&display=swap" rel="stylesheet">
+    <title>Westagram</title>
+</head>
+<body>    
+    <article class="box">
+        <header class="logo">Instagram</header>
+        <input class="id_input" id="id_input" type="text" placeholder="전화번호, 사용자이름 또는 이메일">
```

Comment:
```suggestion
        <input class="id_input" type="text" placeholder="전화번호, 사용자이름 또는 이메일">
```
동일한 id와 class를 부여할 필요는 없어보입니다

Reply:
수정했습니다!